tests/functional/aws-node-sdk/test/object/putObjTagging.js

```diff
@@ -126,5 +130,82 @@ describe('PUT object taggings', () => {
                 done();
             });
         });
+
+        it('should return 403 AccessDenied putting tag with another account',
+        done => {
+            otherAccountS3.putObjectTagging({ Bucket: bucketName, Key:
+              objectName, Tagging: { TagSet: [
+                  {
+                      Key: 'key1',
+                      Value: 'value1',
+                  },
+              ] },
+          }, err => {
+                _checkError(err, 'AccessDenied', 403);
+                done();
+            });
+        });
+
+        it('should return 403 AccessDenied putting tag with a different ' +
+        'account to an object with ACL "public-read-write"',
+        done => {
+            s3.putObjectAcl({ Bucket: bucketName, Key: objectName,
+            ACL: 'public-read-write' }, err => {
+                if (err) {
+                    return done(err);
+                }
+                return otherAccountS3.putObjectTagging({ Bucket: bucketName,
+                  Key: objectName, Tagging: { TagSet: [
+                      {
+                          Key: 'key1',
+                          Value: 'value1',
+                      },
+                  ] },
+                }, err => {
+                    _checkError(err, 'AccessDenied', 403);
+                    done();
+                });
+            });
+        });
+
+        it('should return 403 AccessDenied putting tag to an object ' +
+        'in a bucket created with a different account',
+        done => {
+            async.waterfall([
+                next => s3.putBucketAcl({ Bucket: bucketName, ACL:
+                  'public-read-write' }, err => next(err)),
+                next => otherAccountS3.putObject({ Bucket: bucketName, Key:
+                    objectNameAcl }, err => next(err)),
+                next => otherAccountS3.putObjectTagging({ Bucket: bucketName,
+                      Key: objectNameAcl, Tagging: { TagSet: [
+                          {
+                              Key: 'key1',
+                              Value: 'value1',
+                          },
+                      ] },
+                    }, err => next(err)),
+            ], err => {
+                _checkError(err, 'AccessDenied', 403);
+                done();
+            });
+        });
+
+        it('should put tag to an object in a bucket created with same ' +
+        'account', done => {
+            async.waterfall([
+                next => s3.putBucketAcl({ Bucket: bucketName, ACL:
+                  'public-read-write' }, err => next(err)),
+                next => otherAccountS3.putObject({ Bucket: bucketName, Key:
+                    objectNameAcl }, err => next(err)),
+                next => s3.putObjectTagging({ Bucket: bucketName,
+                      Key: objectNameAcl, Tagging: { TagSet: [
```

Comment:
The `Tagging` object is reused across these tests, perhaps you could define it as a variable.